lib/livebook_web/live/home_live.ex

```diff
@@ -85,7 +86,7 @@ defmodule LivebookWeb.HomeLive do
             </.live_component>
           </div>
 
-          <div class="py-12" data-element="explore-section">
+          <div class="py-12" data-element="explore-section" role="region" aria-label="Explore Section">
```

Comment:
It doesn't matter, but just for consistency :)

```suggestion
          <div class="py-12" data-element="explore-section" role="region" aria-label="explore section">
```